tools-cli/src/main/java/com/hedera/hashgraph/client/cli/options/CollateCommand.java

```diff
@@ -19,7 +19,7 @@
 package com.hedera.hashgraph.client.cli.options;
 
 import com.google.protobuf.InvalidProtocolBufferException;
-import com.hedera.hashgraph.client.cli.helpers.CollatorHelper;
+import com.hedera.hashgraph.client.core.helpers.CollatorHelper;
```

Comment:
Nit: This import line should come after the next one (core.helpers > core.action).